client/src/components/organisms/AdminRequestClientBrowser.tsx

```diff
@@ -0,0 +1,67 @@
+import { gql, useQuery } from "@apollo/client";
+import React, { FunctionComponent, useState } from "react";
+import ClientRequestTable from '../molecules/ClientRequestTable';
+import Request from '../../data/types/request'
+import { Spinner } from "react-bootstrap";
+
+const AdminRequestClientBrowser: FunctionComponent = () => {
+    const [requests, setRequests] = useState<Request[]>([]);
+    const [numRequests, setNumRequests] = useState(0);
+
+    let clientName : string = window.location.href;
+    clientName = clientName.split("client/")[1].replaceAll("-", " ");
```

Comment:
Instead of getting clientName like this, I would use the useParams hook

See here:
https://stackoverflow.com/questions/60998386/using-the-useparams-hook-in-react